src/Jackett.Common/Indexers/Milkie.cs

```diff
@@ -0,0 +1,167 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Jackett.Common.Models;
+using Jackett.Common.Models.IndexerConfig;
+using Jackett.Common.Services.Interfaces;
+using Jackett.Common.Utils;
+using Jackett.Common.Utils.Clients;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NLog;
+
+namespace Jackett.Common.Indexers
+{
+    public class Milkie : BaseWebIndexer
+    {
+        private string TorrentsEndpoint => SiteLink + "api/v1/torrents";
+
+        private new ConfigurationDataAPIKey configData
+        {
+            get => (ConfigurationDataAPIKey)base.configData;
+            set => base.configData = value;
+        }
+
+        public Milkie(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps)
+            : base(name: "Milkie",
+                description: "Milkie.cc (ME) is private torrent tracker for 0day / general",
+                link: "https://milkie.cc/",
+                caps: new TorznabCapabilities(),
+                configService: configService,
+                client: wc,
+                logger: l,
+                p: ps,
+                configData: new ConfigurationDataAPIKey())
+        {
+            Encoding = Encoding.UTF8;
+            Language = "en-us";
+            Type = "private";
+
+            AddCategoryMapping("1", TorznabCatType.Movies, "Movies");
+            AddCategoryMapping("2", TorznabCatType.TV, "TV");
+            AddCategoryMapping("3", TorznabCatType.Audio, "Music");
+            AddCategoryMapping("4", TorznabCatType.PCGames, "Game");
+            AddCategoryMapping("4", TorznabCatType.Console, "Game");
+            AddCategoryMapping("5", TorznabCatType.Books, "Ebook");
+            AddCategoryMapping("6", TorznabCatType.PC0day, "App");
+            AddCategoryMapping("6", TorznabCatType.PCISO, "App");
+            AddCategoryMapping("6", TorznabCatType.PCMac, "App");
+            AddCategoryMapping("6", TorznabCatType.PCPhoneAndroid, "App");
+            AddCategoryMapping("6", TorznabCatType.PCPhoneIOS, "App");
+            AddCategoryMapping("6", TorznabCatType.PCPhoneOther, "App");
+            AddCategoryMapping("6", TorznabCatType.OtherMisc, "App");
+            AddCategoryMapping("7", TorznabCatType.XXX, "Adult");
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+
+            try
+            {
+                var results = await PerformQuery(new TorznabQuery());
+
+                if (results.Count() == 0)
```

Comment:
Use .Any() to avoid enumerating full results.
```suggestion
                if (!results.Any())
```